src/core/edit/edit.cell.view.js

```diff
@@ -49,9 +52,17 @@ export class EditCellView {
 					}
 					else {
 						cell = model.navigation().cell;
+						const code = this.shortcut.keyCode;
+						if (code && code.length > 0) {
+							const reserved = Array.from(this.shortcut.codeMap.values()).some(val => val === code);
+							if (!reserved) {
+								this.memory = code;
```

Comment:
I need a flag to know about starting the input modifications with any key (to add it to the cell value, and to remove it on cancel)